nemo_text_processing/inverse_text_normalization/es/graph_utils.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+# Copyright 2015 and onwards Google, Inc.
```

Comment:
could you remove this line? this file is not copied from english graph_utils.py which has NeMo_DIGIT, plurals, etc defined

Reply:
good point! removed, ty